flow/project.py

```diff
@@ -1171,12 +1171,6 @@ def __init__(self, config=None, environment=None, entrypoint=None):
         self._groups = dict()
         self._register_groups()
 
-        # Enable the use of buffered mode for certain functions
-        try:
-            self._use_buffered_mode = self.config['flow'].as_bool('use_buffered_mode')
-        except KeyError:
-            self._use_buffered_mode = False
```

Comment:
I think this might create problems if there is no 'flow' config entry.

Reply:
@csadorf That's not possible in a `FlowProject` because we monkey-patch the config definition. In that redefinition, it always has a `[flow]` section.

Reply:
Ok, then it should be fine.